doc/faq.rst

```diff
@@ -26,6 +26,41 @@ See :ref:`contributing`. Before wanting to add a new algorithm, which is
 usually a major and lengthy undertaking, it is recommended to start with :ref:`known
 issues <easy_issues>`.
 
+What's the best way to get help on scikit-learn usage?
+--------------------------------------------------------------
+**For general machine learning questions**, please use
+`Cross Validated <http://stats.stackexchange.com>`_ with the ``[machine-learning]`` tag.
```

Comment:
Should we use the tagged URL here?

http://stats.stackexchange.com/questions/tagged/machine-learning